### PR TITLE
Made the outofbounds handler not rely on screen size

### PR DESCRIPTION
**Pull request by BossSloth**

### CODE REVIEW

**Comment by BossSloth** [comment]
You will need to add MMHook to the dependencies in the csproject

**Comment by pdcook** [comment]
I hate to be "that guy" but I'm not able to get this to work in Sandbox or Versus. I disabled all mods in my debug profile except for MapEmbig, UnboundLib, and MMHook. I even removed UnityExplorer.
image
image

**Comment by BossSloth** [comment]
i can't seem reproduce it not working in any way. Did you maybe build the dll wrong? Here is my most recent build of it try if this works for you?
[MapEmbiggener.zip](https://github.com/pdcook/MapEmbiggener/files/7883562/MapEmbiggener.zip)

**Comment by pdcook** [comment]
What the hell, that .zip works for me. What could cause me to "build the dll wrong"?

File: MapEmbiggener.cs [hunk 1]
```diff
@@ -456,26 +457,32 @@ private static void Postfix(Map __instance)
...
         {
             if (instance == null) { yield break; }
-            timerStart = Time.time;
-            rotTimerStart = Time.time;
+            MapPatchStartMatch.timerStart = Time.time;
+            MapPatchStartMatch.rotTimerStart = Time.time;
             while (instance != null && instance.enabled)
             {
-                if (instance != null && (float)Traverse.Create(instance).Field("counter").GetValue() > 2f && instance.size > 1f && (MapEmbiggener.chaosMode || (MapEmbiggener.suddenDeathMode && CountPlayersAlive() <= 2)))
+                if (instance != null && (float)instance.GetFieldValue("counter") > 2f && MapEmbiggener.zoomShrink > 0.3f && (MapEmbiggener.chaosMode || (MapEmbiggener.suddenDeathMode && CountPlayersAlive() <= 2)))
```
Comment (pdcook) [inline MapEmbiggener.cs]: Looking at the original code here, it's clear that I tried to stop Chaos Mode / Sudden Death Mode from shrinking too much, but that never worked. But now, I want to keep it that way; i.e. infinitely zooming in to _force_ the battle to end.

**Comment by pdcook** [comment]
If possible - for Chaos Mode to continue working - could you add the ability for the bounds to be rotated?

**Comment by BossSloth** [comment]
I'll look if it's possible to rotate the bounds tomorrow.

**Pull request merged**